extension/locale/en-US/vimfx.properties

```diff
@@ -125,7 +125,7 @@ pref.hint_chars.title=Hint chars
 pref.hint_chars.desc=
 
 pref.black_list.title=Blacklist
-pref.black_list.desc=URLs where VimFx should automatically enter Ignore mode. You may use * as a wildcard. Example: *example.com*  http://example.org/editor/*.
+pref.black_list.desc=List of URLs where VimFx should automatically enter Ignore mode, delimited by a <space>. You may use * as a wildcard. Example list: *example.com* http://example.org/editor/*
```

Comment:
Looks great! Which is best of these, do you think?

- Your suggestion above.
- `List of URLs where VimFx should automatically enter Ignore mode, delimited by space. You may use * as a wildcard. Example list: *example.com* http://example.org/editor/*`
- `List of URLs where VimFx should automatically enter Ignore mode, delimited by spaces. You may use * as a wildcard. Example list: *example.com* http://example.org/editor/*`
- `Space delimited list of URLs where VimFx should automatically enter Ignore mode. You may use * as a wildcard. Example list: *example.com* http://example.org/editor/*`